### DISCUSSION TITLE
Upgrading from JBrowse 1

### DISCUSSION THREAD

**Discussion by jpbinkley** [GMOD/jbrowse-components#2453] [Q&A]
At the Candida Genome Database (http://www.candidagenome.org) we use JBrowse 1 to display large-scale datasets for multiple Candida species. We would like to upgrade to JBrowse 2. Is there a straightforward way to convert our many configuration files (currently tracks.conf text files and metadata in csv format) into the json format compatible with JBrowse 2? Or will we need to rebuild them from scratch?

Thanks in advance,

Jon Binkley

**Comment by cmdcolin** [comment]
@jpbinkley definitely great to hear! there is not a great path for upgrading right now unfortunately. we started making a basic jb1tojb2 script but it only works on jbrowse 1's trackList.json right now

https://github.com/GMOD/jbrowse-components/pull/2144

if interested, we can probably get that to work on the tracks.conf too. note that not all advanced configurations would carry over also, just since jbrowse 2 is a bit different, but @scottcain has worked on translating the wormbase config with this script

you could try setting up a small instance of jbrowse 2 to get a sense of the functionality, and we'd be happy to help that migration as needed!
